packages/api-reference/src/features/Operation/components/RequestBody.vue

```diff
@@ -111,74 +118,27 @@ const partitionedSchema = computed(() => {
 </template>
 
 <style scoped>
-.request-body-title {
+.request-body-header {
   display: flex;
   align-items: center;
   justify-content: space-between;
-  font-size: var(--scalar-font-size-2);
-  font-weight: var(--scalar-semibold);
-  color: var(--scalar-color-1);
   margin-top: 24px;
   padding-bottom: 12px;
   border-bottom: var(--scalar-border-width) solid var(--scalar-border-color);
   flex-flow: wrap;
 }
-.request-body-title-select {
-  position: relative;
-  height: fit-content;
-  margin-left: auto;
-  font-weight: var(--scalar-regular);
-  display: flex;
-  align-items: center;
-  color: var(--scalar-color-3);
-  font-size: var(--scalar-micro);
-  background: var(--scalar-background-2);
-  padding: 2px 6px;
-  border-radius: 12px;
-  border: var(--scalar-border-width) solid var(--scalar-border-color);
-}
-
-.request-body-title-no-select.request-body-title-select {
-  pointer-events: none;
-}
-.request-body-title-no-select {
-  border: none;
-}
-.request-body-title-no-select.request-body-title-select:after {
-  display: none;
-}
-.request-body-title-select span {
+.request-body-title {
   display: flex;
   align-items: center;
-}
-.request-body-title-select:after {
-  content: '';
-  width: 6px;
-  height: 6px;
-  transform: rotate(45deg) translate3d(0, -3px, 0);
-  display: block;
-  margin-left: 6px;
-  box-shadow: 1px 1px 0 currentColor;
-  margin-right: 5px;
-}
-.request-body-title-select select {
-  border: none;
-  outline: none;
-  cursor: pointer;
-  background: var(--scalar-background-3);
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  opacity: 0;
-  appearance: none;
-}
-.request-body-title-select:hover {
+  gap: 8px;
+  font-size: var(--scalar-font-size-2);
+  font-weight: var(--scalar-semibold);
   color: var(--scalar-color-1);
 }
-.request-body-title-select:has(select:focus-visible) {
-  outline: 1px solid var(--scalar-color-accent);
+.request-body-required {
```

Comment:
how does it feel to baseline align it?

Reply:
I tried it and it looked not great tbh 😅